mailer.rst

```diff
@@ -541,6 +541,97 @@ This makes use of the :ref:`css Twig namespace <mailer-css-namespace>` we create
 earlier. You could, for example, `download the foundation-emails.css file`_
 directly from GitHub and save it in ``assets/css``.
 
+Signing and Encrypting Messages
+-------------------------------
+
+.. versionadded:: 4.4
+
+    The option to sign and/or encrypt messages was introduced in Symfony 4.4.
+
+It's possible to sign and/or encrypt email messages applying the `S/MIME`_
+standard to increase their integrity/security. Both options can be combined (to
+encrypt a signed message and to sign an encrypted message) and they require to
+have the `OpenSSL PHP extension`_ properly installed and configured.
+
+Signing Messages
+~~~~~~~~~~~~~~~~
+
+When signing a message, a cryptographic hash is generated for the entire content
+of the message (including attachments). This hash is added as an attachment so
+the recipient can validate the integrity of the received message. However, the
+contents of the original message are still readable for mailing agents not
+supporting signed messages, so you must also encrypt the message if you want to
+hide its contents::
+
+    use Symfony\Component\Mime\Crypto\SMimeSigner;
+    use Symfony\Component\Mime\Email;
+
+    $email = (new Email())
+        ->from('hello@example.com')
+        // ...
+        ->html('...');
+
+    // if the certificate has a passphrase, pass it as the third argument
```

Comment:
The private-key has a passphrase, not the certificate 😛 the certificate is always fully readable.

Reply:
Oops! Fixed now. Thanks!